### DISCUSSION TITLE
Reconfigure network settings

### DISCUSSION THREAD

**Discussion by yula-21** [Security-Onion-Solutions/securityonion#4509] [Unsupported Versions] [answered]
Hi,
I need to change the network configuration that I set.
Is there a correct way to reconfigure the network on the new version? 
Im new to salt so I don't really understand if the network is configured through an sls file or the basic ifcfg-ens33.
Does anyone know?

**Comment by b4b857f6ee** [comment]
AS far as i know, they didn't really support changing the ip... so you need to reinstall.

**Comment by defensivedepth** [comment, marked as answer]
@yaelf21 Checkout `so-ip-update`

https://docs.securityonion.net/en/2.3/ip.html

**Reply by SanibelJack** [reply]
I have installed 2.3.9 distributed network. I have to change the IP on a sensor. Does the reconfigure network in so-setup work? I ask because I ran the change network but the managersearch sever does not see it now.  Any help would be appreciated

**Reply by defensivedepth** [reply]
@SanibelJack Please open up a new issue, thanks!
